support/src/main/java/se/inera/intyg/common/support/modules/support/facade/ModuleFacadeApi.java

```diff
@@ -33,4 +34,6 @@ public interface ModuleFacadeApi {
     CertificateTextProvider getTextProvider(String certificateType, String certificateTypeVersion);
 
     CertificateMessagesProvider getMessagesProvider();
+
+    String getUtlatandeToInternalModelResponse(Utlatande utlatande) throws ModuleException;
```

Comment:
Suggest renaming to getJsonFromUtlatande(Utlåtande utlåtande) to follow the naming convention.

Reply:
Done.